### PR TITLE
[MOB-2339]   Use latest crux code generator

### PR DESCRIPTION
**Pull request by junggyu-sc**
Use latest crux code generator to address an issue of crux build error with map entry.

### CODE REVIEW

File: protoc-cpp/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.10.2
+1.11.0
```
Comment (codingrhythm) [inline protoc-cpp/version.txt:1]: This should be a patch bump instead of a minor.

Comment (junggyu-sc) [reply protoc-cpp/version.txt:1]: fixed.

Comment (codingrhythm) [reply protoc-cpp/version.txt:1]: @junggyu-sc Did you forget to push your commit?

Comment (junggyu-sc) [reply protoc-cpp/version.txt]: sorry, just updated.

**Review by codingrhythm — APPROVED** [review]

**Pull request merged**